src/reference/asciidoc/amqp.adoc

```diff
@@ -191,8 +192,12 @@ _Optional_.
 
 <25> Tells the `SimpleMessageListenerContainer` how many messages to process in a single transaction (if the channel is transactional).
 For best results it should be less than or equal to the set "prefetch-count".
+Not allowed when 'consumers-per-queue' is set.
 _Optional (Defaults to 1)_.
 
+<26> Indicates that the underlying listener container should be a `DirectMessageListenerContainer` instead of the default `SimpleMessageListenerContainer`.
+Refer to the Spring AMQP Reference Manual for more information.
```

Comment:
Can't we really refer here directly with the link?
Maybe not to `DirectMessageListenerContainer`, but to the entire `Container` paragraph: http://docs.spring.io/spring-amqp/docs/2.0.0.BUILD-SNAPSHOT/reference/html/_reference.html#_container ?